inc/cleantalk-settings.php

```diff
@@ -2679,3 +2684,36 @@ class="apbct-btn-as-link"
                 . __('change email', 'cleantalk-spam-protect') .
             '</button>)';
 }
+
+/**
+ * Implementation of service_update_local_settings functionality
+ */
+add_action('apbct_before_returning_settings', 'apbct__send_local_settings_to_api');
+
+function apbct__send_local_settings_to_api($settings)
+{
+    if (empty($settings) || !is_array($settings)) {
+        return;
+    }
```

Comment:
I think this condition is redundant $settings can not be empty.

Reply:
theoretically, this function can be used in some other situations, so I'm adding checks so that the function works normally in the future.

Reply:
ok, removed